students/M_Sunday/lesson01/break_me.py

```diff
@@ -0,0 +1,18 @@
+# lesson 01 task 1
+
+def call_except(type):
+    if type == 'NameError':
+        print(x)
+    elif type == 'TypeError':
+        a = 'bye'
+        str = 'hello'
+        print(a * str)
+    elif type == 'SyntaxError':
+        #print(,)
+        pass
+    elif type == 'AttributeError':
+        text = 'cool'
+        text.atterror
+
+call_except(type = 'AttributeError')
```

Comment:
be careful about overriding built-in functions in python 🙂 

these usually have special syntax highlighting and if your editor doesn't highlight, github will. notice the highlight on `type` var, it's a built-in function.